docs/source/conf.py

```diff
@@ -18,8 +18,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'bravado'
-copyright = u'2013, Digium, Inc.; 2014-2015, Yelp, Inc'
+project = 'bravado'
+copyright = '2013, Digium, Inc.; 2014-2015, Yelp, Inc'
```

Comment:
Add Cisco here :)

Reply:
we already have it defined in setup.py 
https://github.com/Cloudlock/bravado/blob/master/setup.py#L14
Do I still need to change here as well?